manifests/t/the-sz/York/1.66/the-sz.York.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.York
+PackageVersion: 1.66

```

Comment:
```suggestion
PackageVersion: 1.65
```

Reply:
It's now fixed, the setup had the old version number...